1.2.0-alpha.1/Classes/Directions.html

```diff
@@ -0,0 +1,1433 @@
+                    <a class="token" href="#/s:16MapboxDirections0B0C7Sessiona"><span class="token-name">Session</span><span class="token-icon" /></a>
+                    </code>
+                  </div>
+                  <div class="height-container">
+                    <section class="section">
+                      <div class="abstract">
+                        <p>A tuple type representing the direction session that was generated from the request.</p>
+
+                      </div>
+                      <div class="declaration">
+                        <h4>Declaration</h4>
+                        <div class="language keyline-blue">
+                          <p  style="background: rgba(65, 131, 196, 0.15);" class="aside-title text-blue">Swift</p>
+                          <pre class="highlight swift"><code><span class="kd">public</span> <span class="kd">typealias</span> <span class="kt">Session</span> <span class="o">=</span> <span class="p">(</span><span class="nv">options</span><span class="p">:</span> <span class="kt"><a href="../Classes/DirectionsOptions.html">DirectionsOptions</a></span><span class="p">,</span> <span class="nv">credentials</span><span class="p">:</span> <span class="kt">DirectionsCredentials</span><span class="p">)</span></code></pre>
+
+                        </div>
+                      </div>
+                      <div>
+                        <h4>Parameters</h4>
+                        <table class="graybox">
+                          <tbody>
+                            <tr>
+                              <td>
+                                <code>
+                                  options
+                                </code>
+                              </td>
+                              <td>
+                                <div>
+                                  <p>A <code><a href="../Classes/DirectionsOptions.html">DirectionsOptions</a></code> object representing the request parameter options.</p>
+                                </div>
+                              </td>
+                            </tr>
+                            <tr>
+                              <td>
+                                <code>
+                                  credentials
+                                </code>
+                              </td>
+                              <td>
+                                <div>
+                                  <p>A object containing the credentials used to make the request.</p>
+                                </div>
+                              </td>
+                            </tr>
+                          </tbody>
+                        </table>
+                      </div>
+                      <div class="show-on-github">
+                        <a href="https://github.com/mapbox/mapbox-directions-swift/tree/v1.0.0/Sources/MapboxDirections/Directions.swift#L78">
```

Comment:
All the links are pointing to v1.0.0 instead of v1.2.0-alpha.1. This is probably because you haven’t fetched all the tags and also because you haven’t tagged v1.2.0-alpha.1 yet. The documentation generation step needs to happen after you merge #501 and tag the release, so you’ll need to regenerate the docset.

Reply:
Yep, tags are not there yet. Will do that, thanks.